kafka/consumer_test.go

```diff
@@ -46,13 +51,38 @@ func TestConsumerAPIs(t *testing.T) {
 		t.Fatalf("%s", err)
 	}
 
+	closedConsumer, err := NewConsumer(&ConfigMap{
+		"group.id":                 "gotest",
+		"socket.timeout.ms":        10,
+		"session.timeout.ms":       10,
+		"enable.auto.offset.store": false, // permit StoreOffsets()
+	})
+	if err != nil {
+		t.Fatalf("%s", err)
+	}
+	closedConsumer.Close()
+
+	var nilConsumer *Consumer = nil
+
 	t.Logf("Consumer %s", c)
+	t.Logf("Consumer %s", closedConsumer)
```

Comment:
For these to be helpful you need to have the log differentiate them: "closedConsumer %s" and "nilConsumer %s" (still don't think we need to support nils).